### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kinire98**
A little update in the README for it to be more accurate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ This example opens a serial port and echoes back everything that is read.
 ```rust
 use serial2::SerialPort;
 
-// On Windows, use something like "COM1".
+// On Windows, use r"\\\\.\\COM1" or just "COM1".
```
Comment (de-vri-es) [inline README.md:41]:
```suggestion
// On Windows, use something like "COM1".
// For COM ports above COM9, you need to use the win32 device namespace, for example "\\.\COM10" (or "\\\\.\\COM10" with string escaping).
// For more details, see: https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file?redirectedfrom=MSDN#win32-device-namespaces
```

Comment (de-vri-es) [reply README.md:41]: Thanks for the PR. I like to idea of explaining the win32 device namespace, but I would make it a bit more explicit with regards to escaping. And I'd like to add a link to the microsoft documentation.

**Comment by de-vri-es** [comment]
Applied the tweak and merged. Thanks again for the PR!

**Comment by de-vri-es** [comment]
Released in 0.1.8. Also added it to the docs in `lib.rs` and a similar note for `SerialPort::open()`.

**Comment by kinire98** [comment]
You're welcome! The info you added is very interesting, didn't know that!

**Pull request merged**